lib/lockfile.js

```diff
@@ -65,7 +65,13 @@ async function getNewLockfile ({ packageJson, lock, isNpm, repositoryTokens }) {
     })
       .then(result => {
         jobCountByServer[nextServer]--
-        return result
+        if (result instanceof Error) {
```

Comment:
have you tried this? At this point result *might* just be a POJO, not an `instanceof Error` since it went over HTTP once.

Reply:
just tried it. it is an instance of error that was catched in the exec-server and then returned.  

But I realized that the npm & yarn errors are not just  `Comand Failed` but also `an unexpected error occured`. so it needs a bit more work on the exec-server